src/index.html

```diff
@@ -6,13 +6,60 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="container"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="iMac"
+        class="container__photo"
+      />
+      <h3 class="container__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="container__code">Product code: 195434</p>
+      <div class="container__feedback">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <article>Reviews: 5</article>
+      </div>
+      <div class="container-amount">
+        <p class="container-amount__price">Price:</p>
+        <p class="container-amount__sum">$2,199</p>
+      </div>
+      <a
+        href="#"
+        data-qa="hover"
+        class="container__button"
+      >
+        Buy
+      </a>
+    </div>
```

Comment:
It's better to use more semantic tags for the structure of your card. For example, you could use `<article>` for the card container, `<figure>` for the image, `<figcaption>` for the title, `<p>` for the code and `<section>` for the feedback and price. This will make your code more accessible and easier to understand.